doc/code/sf.rst

```diff
@@ -3,7 +3,5 @@ sf
 
 .. currentmodule:: strawberryfields
 
-.. automodapi:: strawberryfields
-    :no-heading:
-    :include-all-objects:
-    :no-inheritance-diagram:
+.. automodule:: strawberryfields
+   :members: about, cite, version
```

Comment:
Now it looks like this, except without LocalEngine (it's identical to Engine, so I though it'd just be confusing to the user).

`sf.math` is not documented here because of the complexities of how Sphinx documents module-level variables. As far as I understand they cannot have real introspectable Python docstrings in general, instead Sphinx parses the source code and interprets a string literal following a variable as a docstring. This is however not inherited by the imported version of the variable in another module, so it would just show the docstring of the `SimpleNamespace` class.